plugins/flutter_aepedge/README.md

```diff
@@ -102,10 +103,13 @@ public class MainApplication extends FlutterApplication {
 In your Flutter application, import the Edge extension as follows:
 ```dart
 import 'package:flutter_aepedge/flutter_aepedge.dart';
+import 'package:flutter_aepedge/flutter_aepedge_data.dart';
```

Comment:
Why is this required? I see flutter_aepedge.dart imports the data too - https://github.com/adobe/aepsdk_flutter/blob/staging/plugins/flutter_aepedge/lib/flutter_aepedge.dart